From 574db805d50abde5b4dd6a698846224b508e9fd4 Mon Sep 17 00:00:00 2001
From: AKASHI Takahiro <takahiro.akashi@linaro.org>
Date: Tue, 17 Nov 2020 09:27:16 +0900
Subject: [PATCH 1/1] dfu: simplify the dependencies of DFU_TFTP

Since CONFIG_UPDATE_COMMON always selects CONFIG_DFU_WRITE_ALT, we can
drop the latter from dependencies of CONFIG_DFU_TFTP.

Fixes: 3149e524fc1e ("common: update: add a generic interface for FIT
       image")
Signed-off-by: AKASHI Takahiro <takahiro.akashi@linaro.org>
Reviewed-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 drivers/dfu/Kconfig | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/dfu/Kconfig b/drivers/dfu/Kconfig
index b7427fc4f0..121dc54f54 100644
--- a/drivers/dfu/Kconfig
+++ b/drivers/dfu/Kconfig
@@ -20,9 +20,8 @@ config DFU_WRITE_ALT
 
 config DFU_TFTP
 	bool "DFU via TFTP"
-	select DFU_WRITE_ALT
-	select DFU_OVER_TFTP
 	select UPDATE_COMMON
+	select DFU_OVER_TFTP
 	help
 	  This option allows performing update of DFU-managed medium with data
 	  sent via TFTP boot.
-- 
2.29.2

